Gill_Kyle/details.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="app_base.htm" />
+
+<%block name="title">
+    <h1>${ page_title }</h1>
+</%block>
+
+<%block name="content">   
+    <div class="col-sm-8">
+        <div class="col-sm-2">
+        % for i in product_images:
```

Comment:
I am not sure as to how this works; but Hammy approached this the same way. How does the javascript know which image it has selected?

Reply:
The selectors are all in the javascript file, it means it relies on the html nesting of the tags here so it can find when an image is being hovered over but there doesn't have to be any other code here in the template.